packages/terra-menu/src/Menu.jsx

```diff
@@ -53,12 +53,18 @@ const propTypes = {
    * Otherwise, the menu will display without an arrow and right aligned.
    */
   isArrowDisplayed: PropTypes.bool,
+  /**
+   * Header Title for main-menu(first-tier).
+   * Header Title will only be visible if the main-menu contains at least one sub-menu.
+   */
+  headerTitle: PropTypes.string,
 };
 
 const defaultProps = {
   isArrowDisplayed: false,
   isOpen: false,
   contentWidth: '240',
+  headerTitle: '',
```

Comment:
we could set this as Undefined

Reply:
There is a length check on headerTitle, setting the default value to `undefined` will result in a error.